### DISCUSSION TITLE
WMTS endpoint preview feature in the data API?

### DISCUSSION THREAD

**Discussion by ZirconCode** [microsoft/PlanetaryComputer#196] [Q&A] [answered]
I saw a preview feature for tiling [here ](https://planetarycomputer.microsoft.com/docs/reference/data/): `/api/data/v1/item/tiles/{z}/{x}/{y}.{format}`. This looks very much like a WMTS endpoint for something like leaflet.js for example. Is this the idea, because we would be very interested in it, that would be amazing. What confuses me a little is that a `STAC Item ID` is required, and it seems I need to provide a specific tile id and not a band (and perhaps time) like ex. `B04`, but then the `x,y,z` coordinates make no sense to me. I've tried to explore this for the Sentinel 2 dataset, what is the idea behind this? Thanks!

**Comment by TomAugspurger** [comment, marked as answer]
https://planetarycomputer.microsoft.com/docs/quickstarts/using-the-data-api/ has some documentation that might be helpful.

**Comment by ZirconCode** [comment]
Ah thanks, awesome, that's what I was looking for! Do you know if we can use this endpoint directly to display tiles to our users in their browsers, are there rate or other restrictions we should be aware of other than that it is a preview feature?

**Reply by TomAugspurger** [reply]
No rate limits, AFAIK.

If you're developing your own expressions / rendering options, note that [TiTiler](https://developmentseed.org/titiler/) and its dependencies, the projects this builds on, recently made some breaking changes to its expression system. See https://github.com/stac-utils/titiler-pgstac/blob/master/CHANGES.md?plain=1#L55-L63 for more.

Let us know if this ends up being useful, and if you run into any issues!

**Reply by ZirconCode** [reply]
Amazing thank you, will do!
